coml/core.py

```diff
@@ -229,3 +231,14 @@ def output_sanity_check(
         if "CORRECT" in last_line.upper():
             return True, reason
         raise ValueError("Unable to parse the response.")
+    
+    def vis_check(self, request:str, previous_code:str, svg_string: str, variable_descriptions: dict[str, str],source):
```

Comment:
`visualization_check` might be a better name as it's more consistent with other names.